src/components/BankAccounts/BankAccounts.spec.js

```diff
@@ -0,0 +1,14 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import BankAccounts from '.';
+
+describe('Bank Accounts', () => {
+	it('should render correctly', () => {
+		const accountName = 'Konto A';
+		const bankName = 'Bank XY';
+		const accountValue = '21983€';
+		render(<BankAccounts>{(accountName, bankName, accountValue)}</BankAccounts>);
+		const bankAccounts = screen.getByText(accountName, bankName, accountValue);
+		expect(bankAccounts).toBeInTheDocument();
+	});
```

Comment:
Currently, your test only checks if the first bank account is rendered. I think you should check if every bank account is rendered according to how your component is currently implemented. If you want to change your component like I suggested in my other comment, this test may work. If this sounds confusing, feel free to ask me and we can speak about this tomorrow. 😅